src/components/to-do/To-Do.component.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './To-Do.component.css';
+import Task from "./subElement/Task.component";
+
+export default class ToDo extends Component {
+  constructor(props) {
+    super(props);
+    this.addTask = this.addTask.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.taskFinished = this.taskFinished.bind(this);
+    this.editTask = this.editTask.bind(this);
+    this.deleteTask = this.deleteTask.bind(this);
+    this.state = {
+      newTask: '',
+      tasksList: []
+    }
+  }
+  addTask() {
+    if (this.state.newTask) {
+      this.state.tasksList.push({
+        task: this.state.newTask,
+        state: false
+      });
+      this.setState(state => ({
+        tasksList: state.tasksList,
```

Comment:
[ ...state.tasksList, 
     { task: this.state.newTask,
        state: false
      }
]